worlds/soe/__init__.py

```diff
@@ -230,6 +318,9 @@ def modify_multidata(self, multidata: dict):
             payload = multidata["connect_names"][self.world.player_name[self.player]]
             multidata["connect_names"][self.connect_name] = payload
 
+    def get_required_client_version(self):
```

Comment:
generally the max(super()) call is preferred, as the parent version number may be raised when features are removed ffrom the network layer.

Reply:
pushed over it. should be fine now?